client/scss/components/_listing.scss

```diff
@@ -783,4 +788,4 @@ table.listing {
       }
     }
   }
-}
+}
```

Comment:
Please remove these linting changes from the pull request. 

While they may be formatted by your editor, they are not needed as part of this proposed change and just add a bit of noise.

Reply:
Plus. They look like they are flagging as failing the CI.

Reply:
yes, I've removed the prettier formatting
...BUT, I did formatting in the first place because CI told me to do that, as it was failing due to that only.

PS: it's still failing, because maybe previously someone had merged the code without checking all this stuff.
PSS: linting is not the only error if you explore the code more. there are many bad practices.

I suggest keeping it as it is for now. And raising an issue to FIX this problem. Maybe I'll only do it.

Reply:
Ok. It seems to be running fine now (CI) without all those other CSS changes so not sure what happened without digging deeper sorry